plugins/jobtype/jobtypes/teradataToHdfs/plugin.properties

```diff
@@ -0,0 +1,5 @@
+mr.listener.visualizer=false
```

Comment:
"lib" and "libjar" is different where libjar will be put into the distributed cached used by mapper/reducer by Teradata connector. "lib", just like other job types, will be added into the classpath of jvm that invokes Teradata connector.